systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -83,7 +83,7 @@ public void testDeployConnectClusterViaTemplate() {
         String deploymentName = clusterName + "-connect";
         oc.waitForDeployment(deploymentName, 1);
         testDockerImagesForKafkaConnect();
-        oc.deleteByName("cm", clusterName);
+        oc.deleteByName("deployment", deploymentName);
```

Comment:
What is the purpose of this? Why do we delete the Deployment? If the CO is running, it might start recreating these resources an slow down the tests. If it is not running, the Deployment was never created.

I assume that the original version of this test deleted the cluster ConfigMap which triggered the deletion of the deployment by the CO. In such case, you need to change the `oc.deleteByName("cm", clusterName);` to something more like this: `oc.deleteByName("KafkaConnect", clusterName);`. That should trigger the CO to delete the deployment.

Reply:
awesome @scholzj , that's what I wanted to ask you because I didn't find this in the documentation.